lib/multistream.js

```diff
@@ -0,0 +1,138 @@
+'use strict'
+
+const metadata = Symbol.for('pino.metadata')
+
+const defaultLevels = {
```

Comment:
We should be able to pull these from `lib/levels` now, right?

Reply:
done